pkg/virt-api/api.go

```diff
@@ -786,7 +786,7 @@ func (app *virtAPIApp) Compose() {
 func (app *virtAPIApp) ConfigureOpenAPIService() {
 	config := openapi.CreateConfig()
 	config.GetDefinitions = v12.GetOpenAPIDefinitions
-	spec, err := builderv3.BuildOpenAPISpecFromRoutes(restfuladapter.AdaptWebServices(restful.RegisteredWebServices()), config)
+	spec, err := builder.BuildOpenAPISpecFromRoutes(restfuladapter.AdaptWebServices(restful.RegisteredWebServices()), config)
```

Comment:
Why do we roll back?

Reply:
Once I bumped the k8s version there was a version inconsistency, and `config` (of type `*common.Config`, created by `openapi.CreateConfig()`) was no longer of the appropriate type (`*common.OpenAPIV3Config`).

They moved the config to the v3 API.

Anyways per your question - the reason is I thought it would be the least change to move to the old function to keep consistency, as changing to builderv3 will result in other bigger changes.

But if you think that we should bump swagger to v3 as well then we can do that.